src/Android/Xamarin.Android/Samples/Symbology/CustomDictionaryStyle/CustomDictionaryStyle.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArcGISRuntimeXamarin.Samples.CustomDictionaryStyle
+{
+    [Activity(ConfigurationChanges = Android.Content.PM.ConfigChanges.Orientation | Android.Content.PM.ConfigChanges.ScreenSize)]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Custom dictionary style",
+        "Symbology",
+        "Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.",
+        "")]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData("751138a2e0844e06853522d54103222a")]
+    public class CustomDictionaryStyle : Activity
+    {
+        // Hold references to the MapView.
+        private MapView _myMapView;
+
+        // The custom dictionary style for symbolizing restaurants.
+        private DictionarySymbolStyle _restaurantStyle;
+
+        // Uri for the restaurants feature service.
+        private readonly Uri _restaurantUri = new Uri("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0");
+
+        // Path for the restaurants style file.
+        private readonly string _stylxPath = DataManager.GetDataFolder("751138a2e0844e06853522d54103222a", "Restaurant.stylx");
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Custom dictionary style";
+
+            CreateLayout();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            try
+            {
+                // Open the custom style file.
+                _restaurantStyle = await DictionarySymbolStyle.CreateFromFileAsync(_stylxPath);
+
+                // Create a new map with a streets basemap.
+                Map map = new Map(Basemap.CreateStreetsVector());
```

Comment:
For performance with some network configurations, I recommend using the raster basemap on Android and Forms